src/SparqlLua.php

```diff
@@ -4,27 +4,39 @@
 
 namespace ProfessionalWiki\SPARQL;
 
+use InvalidArgumentException;
+use MediaWiki\MediaWikiServices;
 use Scribunto_LuaLibraryBase;
 
 class SparqlLua extends Scribunto_LuaLibraryBase {
 
 	/**
-	 * @return mixed[]
+	 * @return array<mixed>
```

Comment:
If you're forced to add types when they don't really add value, then you can instead add them to the PHPStan/Psalm baselines to ignore them.
```
make stan-baseline
make psalm-baseline
```

Reply:
Yes. It was forced by stan. Saying
"Method ProfessionalWiki\SPARQL\SparqlLua::register() return type has no value type specified in iterable type array.  

         💡 See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type "
         
![image](https://github.com/ProfessionalWiki/SPARQL/assets/15813104/2838e479-4a5f-4daf-9744-b33356f869fc)
@JeroenDeDauw